src/main/groovy/pl/allegro/tech/build/axion/release/infrastructure/git/GitRepository.groovy

```diff
@@ -14,14 +14,15 @@ import org.eclipse.jgit.revwalk.RevCommit
 import org.eclipse.jgit.revwalk.RevSort
 import org.eclipse.jgit.revwalk.RevWalk
 import org.eclipse.jgit.transport.*
-import org.gradle.api.logging.Logger
+import org.slf4j.LoggerFactory
 import pl.allegro.tech.build.axion.release.domain.scm.*
+import org.slf4j.Logger
 
 import java.util.regex.Pattern
 
 class GitRepository implements ScmRepository {
 
-    private final Logger log;
+    private final Logger log = LoggerFactory.getLogger(GitRepository.getClass())
```

Comment:
Why not just `GitRepository.class`? Or as it is Groovy just `LoggerFactory.getLogger(GitRepository)`?